doc/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Changes in Version 4.10.0
+-------------------------
+
+- Provisional **(BETA)** support for a new Binary BSON subtype used for efficient storage and retrieval of vectors: densely packed arrays of numbers, all of the same type.
```

Comment:
Suggest adding: "New methods :meth:`bson.binary.Binary.from_vector` and
  :meth:`bson.binary.Binary.as_vector`"

Reply:
We should also call out which subtype it is, e.g. "BSON binary subtype 4" elsewhere in the changelog.